assets/lang/pt_BR.ini

```diff
@@ -677,7 +677,7 @@ Texture Replacement = Substituição das texturas
 Toggle Fullscreen = Alternar pra Tela cheia
 Toggle mode = Alternar modo
 Triangle = Triângulo
-Unthrottle = Desacelerador
+Fast-forward = Desacelerador
```

Comment:
I don't speak portuguese/brasialian myself but this seem kinda the same mistake of italian translation, might worth a check if you know someone that speak it.